ui/css/oae.signup.css

```diff
@@ -0,0 +1,310 @@
+/*!
+ * Copyright 2014 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+main > h3 {
+    border-bottom: 1px #EEEEEE solid;
+    margin-left: 20px;
+    margin-right: 20px
+}
+
+#signup-options-container {
+    display: -webkit-flex;
+    display:  -ms-flexbox;
+    display:         flex;
+    -webkit-justify-content: center;
+              -ms-flex-pack: center;
+            justify-content: center;
+    margin: 50px auto;
+}
+
+#signup-options-container > div {
+    margin-right: 5%;
+}
+
+#signup-options-container > div:last-child {
+    margin-right: 0;
+}
+
+#signup-options-container h5 {
+    color: #888;
+    font-size: 15px;
+    font-weight: normal;
+    margin-bottom: 35px;
+    text-align: left;
+}
+
+#signup-options-external {
+    min-width: 225px;
+}
+
+#signup-options-external > ul.nav {
+    margin: 0 auto;
+}
+
+#signup-options-external > ul.nav > li {
+    margin-bottom: 10px;
+}
+
+#signup-options-divider-vertical {
+    border-left: solid 1px #ccc;
+    display: -webkit-flex;
+    display:  -ms-flexbox;
+    display:         flex;
+    -webkit-flex-direction: column;
+        -ms-flex-direction: column;
+            flex-direction: column;
+    -webkit-justify-content: center;
+              -ms-flex-pack: center;
+            justify-content: center;
+    width: 0;
```

Comment:
Added a comment explaining why no width is better than some width:

`/* For the outer container, we only need the border, so any inner width will only uncenter it */`